infrastructure/smart-contracts/contracts/drops/TinyKeysAirdrop.sol

```diff
@@ -161,6 +162,9 @@ contract TinyKeysAirdrop is Initializable, AccessControlUpgradeable {
                 
                 // Stake the keys
                 PoolFactory2(poolFactoryAddress).stakeKeysAdmin(poolAddress, stakeKeyIds, owner);
+
+                // Emit the event
+                emit AirdropSegmentStakeComplete(owner, poolAddress, startingKeyId, endingKeyId);
```

Comment:
I think this should be emitting the startKey of the new keys that got auto staked, not of the ones we processed

` emit AirdropSegmentStakeComplete(owner, poolAddress, stakeKeyIds[0], stakeKeyIds[stakeKeyIds.length-1]);`

Reply:
Done, great catch!